### ISSUE TITLE
Build Clean Command

### ISSUE THREAD

**Issue by mkummer225** [continuedev/continue#2053]
### Validations

- [X] I believe this is a way to improve. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) that requests the same enhancement

### Problem

Would be nice to have a build-clean command that clears out previous build assets, node_modules, etc before starting a debug session

### Solution

Create a new runnable command that clears the previous build assets, node_modules, etc then runs install deps command, then builds and starts a debug session

**Comment by sestinj** [comment]
Here's a tasks for the uninstalling part of this: https://github.com/continuedev/continue/commit/05f08d380099a216e2fd0440c525b533108a041b

My preference would be to keep it separate from install-all-dependencies and the start of a debug session, as these are all expensive operations

**Comment by mkummer225** [comment]
This will definitely work - don't mind the 1 extra step; basically saw the value when finishing up with PR work and wanting to clear everything out to ensure the build works from a clean slate (when adding dependencies for ex.)
